src/app/shared/services/paginator.service.ts

```diff
@@ -0,0 +1,29 @@
+import {Injectable} from '@angular/core';
```

Comment:
The paginator service can be part of the paginator package and not in shared service